### ISSUE TITLE
Example App Crashed

### ISSUE THREAD

**Issue by Meekohi** [vadymmarkov/Tuner#53]
Example App crashes when trying to open the Microphone on Simulator.

**Comment by vadymmarkov** [comment]
It's not possible to use the app on Simulator, try to run it on a device.

**Comment by Meekohi** [comment]
Yup finally figured that out. Might be nice to have a useful error message there :) AudioKit has a kind of slick way of handling this (grabbing the "real device mic" if available but falling back to OSX audio for simulator/debugging), see: https://github.com/AudioKit/AudioKit/tree/f99e45103f42b02f995df7271264828d9d3e1dc1/AudioKit/Common/Internals/EZAudio
